services/jenkins/jenkins-common.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const Joi = require('joi')
+
+const queryParamSchema = Joi.object({
+  disableStrictSSL: Joi.equal(''),
```

Comment:
Although I usually try to refrain from using negated variable names, naming this with the `disable` seemed to be the best approach given the default behavior we want here and the way boolean query params are handled (empty string when set, undefined when not set).

With this approach, a user will have to explicitly include the `disableStrictSSL` query param:
https://shields-staging-pr-3302.herokuapp.com/jenkins/s/https/jenkins-oss.wixpress.com/job/detox-android-56-master.svg?disableStrictSSL
![](https://shields-staging-pr-3302.herokuapp.com/jenkins/s/https/jenkins-oss.wixpress.com/job/detox-android-56-master.svg?disableStrictSSL)

As the default behavior (no query param provided) will still enforce strict ssl, which is why the below badge will fail (there's a cert chain issue with the target jenkins instance)
https://shields-staging-pr-3302.herokuapp.com/jenkins/s/https/jenkins-oss.wixpress.com/job/detox-android-56-master.svg
![](https://shields-staging-pr-3302.herokuapp.com/jenkins/s/https/jenkins-oss.wixpress.com/job/detox-android-56-master.svg)